### ISSUE TITLE
Sacral measurements not saving/appearing

### ISSUE THREAD

**Issue by Lariabeth** [spawaskar-cora/cora-docs#535]
The original analyst entered measurements for the sacrum.  When I peer reviewed, no measurements were showing up and I entered measurements in the review screen.  When the analyst re-took measurements during the peer review process, these measurements did not save and are still not showing up.  This is peculiar and I haven't noticed this with other cases or bones.  I've also heard others mentioning issues with measurements being accepted in the review screen and not pushing over into the measurement screen.

**Comment by clegarde** [comment]
I just checked a sacrum that was recently added (CIL 2017-310-X-5104--903) for Ploesti project and it has a measurement saved, as well as the "Measured" check box being checked. 
![Sacrum_measurements](https://user-images.githubusercontent.com/7350474/64129736-970afb80-cd83-11e9-89b1-2fd7172cae3b.png)
This measurement shows up in the Review screen. I also added a measurement under the review column and clicked save. Both are still showing. 
![Sacrum_measurements2](https://user-images.githubusercontent.com/7350474/64129783-e3eed200-cd83-11e9-9244-5a7d966ba2f5.png)
Could be a further downstream (clicking Accept as the inventorier?) or other issue.
